qa/httpCalls/transaction/sendCoinsToAddress.py

```diff
@@ -16,7 +17,10 @@ def sendCoinsToAddress(sidechainNode, address, amount, fee, api_key = None):
         response = sidechainNode.transaction_sendCoinsToAddress(request, api_key)
     else:
         response = sidechainNode.transaction_sendCoinsToAddress(request)
-    return response["result"]["transactionId"]
+    if "error" in response:
+        raise Exception(response['error']['description'], response['error']['detail'])
```

Comment:
Could you tell me please, why don't we use case class here?

Reply:
what do you mean? we are in python here, do we have case class also here?